### PR TITLE
Feature/dx 976 logourl

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-order-purchase.md [hunk 1]
```diff
@@ -205,6 +206,7 @@ Content-Type: application/json
...
 | {% icon check %} | └─➔&nbsp;`termsOfServiceUrl`      | `string`     | {% include field-description-termsofserviceurl.md %}                                                                                                                                                                                                                                                     |
+| {% icon check %} | └─➔&nbsp;`logoUrl`      | `string`     | If the configuration is activated and you send in a `logoUrl`, then the Swedbank pay logo is replaced with the logo sent in. If you do not send in a logoUrl, then no logo and no text is shown. If the configuration is deactivated, and you send in a logoUrl, this is not handled.                                                                                                                                                                                  |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md:209]:
```suggestion
| {% icon check %} | └─➔&nbsp;`logoUrl`      | `string`     | With permission and activation on your contract, sending in a `logoUrl` will replace the Swedbank Pay logo with the logo sent in. If you do not send in a `logoUrl`, then no logo and no text is shown. Without permission or activation on your contract, sending in a `logoUrl` has no effect.                          |
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
There should be some screenshots as well, no?

**Review by asbjornu — APPROVED** [review]

File: _includes/payment-order-other-features.md [hunk 2]
```diff
@@ -1387,6 +1435,8 @@ principle](https://en.wikipedia.org/wiki/Robustness_principle)." %}
 [payment-resource]: #payments-resource
 [settlement-and-reconciliation]: #settlement-and-reconciliation
 [split-settlement]: #split-settlement
+[swedbankpay-logo-payment]: /assets/screenshots/logourl/logourl-swedbank.png
+[swapped-logo-payment]: /assets/screenshots/logourl/logourl-swapped.png
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:1439]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6039190)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6039190&bid=19788691)

File: _includes/payment-order-other-features.md [hunk 3]
```diff
@@ -1387,6 +1435,8 @@ principle](https://en.wikipedia.org/wiki/Robustness_principle)." %}
 [payment-resource]: #payments-resource
 [settlement-and-reconciliation]: #settlement-and-reconciliation
 [split-settlement]: #split-settlement
+[swedbankpay-logo-payment]: /assets/screenshots/logourl/logourl-swedbank.png
```
Comment (payex-dev) [inline _includes/payment-order-other-features.md:1438]: Codacy found an issue: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6039190)

**Pull request merged**